website/content/en/status/report-2025-01-2025-03/srcmgr.adoc

```diff
@@ -0,0 +1,15 @@
+=== srcmgr
```

Comment:
I would try to make it looking more as a title. Only one word with lowercase initial looks ugly to me.
Looking at https://www.freebsd.org/administration/ I suggest the following:

```suggestion
=== Source Management Team
```